### PR TITLE
Philips HUE (Independent Publisher)

### PR DESCRIPTION
**Pull request by tposzytek**
This is the first release of Philips HUE connector. I will be updating it with new actions.

---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [X] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [X] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [X] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [X] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

### Screenshots:

Connector validation

Cloud flow successfull execution

Connector CLI validation

### CODE REVIEW

**Comment by Amjed-Ayoub** [comment]
@tposzytek Thank you for all your hard work with us. We really appreciate what you add to our Independent Publisher program.

**Comment by Amjed-Ayoub** [comment]
Dear partner,

We apologize for the delay you might be experiencing while we are working on a high volume of connector requests. Rest assured that we are doing everything we can to address this volume. We are taking measures like improving our processes and hiring additional team members to guarantee resolving your connector requests in a timely manner. 

Please let us know if you have any questions in the meantime.

File: independent-publisher-connectors/Philips HUE/apiProperties.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+              "value": "https://api.meethue.com/v2/oauth2/token"
+            }
+          }
+        }
+      },
+      "apiKey": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "HUE application key",
+          "description": "The API Key that is returned after performin PUT and POST authorization calls.",
```
Comment (v-eisljami) [inline independent-publisher-connectors/Philips HUE/apiProperties.json:33]: All descriptions in apiProperties.json file should be free from grammatical errors. Please review them and fix them.

File: independent-publisher-connectors/Philips HUE/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,2176 @@
...
+                    "properties": {
+                      "description": {
+                        "type": "string",
+                        "x-nullable": true,
+                        "description": "A human-readable explanation specific to this occurrence of the problem.",
+                        "title": "Error description"
+                      }
+                    }
+                  },
+                  "description": "Errors that occured during the call."
```
Comment (v-eisljami) [inline independent-publisher-connectors/Philips HUE/apiDefinition.swagger.json:61]: All descriptions in the apiDefinition.swagger.json file should be free from grammatical errors. This also applies to summaries, titles, etc. Please review them and fix them.

**Review by v-eisljami — COMMENTED** [review]
Please remove these images from your PR. These are not needed in the PR itself.

**Comment by tposzytek** [comment]
I have reviewed all the files and removed issues you pointed out.

**Review by v-eisljami — APPROVED** [review]

**Comment by v-eisljami** [comment]
[[certify-connector]]

**Review by raghavthind2005 — APPROVED** [review]

**Comment by Amjed-Ayoub** [comment]
Hi @tposzytek, I hope you are doing well. I want to follow up with you and let you know that your connector is in production deployment. this process will complete within 4-6 weeks.
Please let me know if you have any questions.
Thank you very much.

**Comment by Amjed-Ayoub** [comment]
Hi @tposzytek, I hope you are doing well.
Apologies for the delay in the deployment process. There was an issue with the deployment train that prevented us from completing your connector deployment. We are working to fix this issue; we will keep you posted.
Please let me know if you have any questions.
Thank you so much for your patience and understanding.

**Review by Amjed-Ayoub — APPROVED** [review]

**Pull request merged**